### PR TITLE
Remove import of ert.JobStatus

### PR DESCRIPTION
**Pull request by jonathan-eq**
Blocked by https://github.com/equinor/ert/pull/8389

### CODE REVIEW

File: everest/suite.py [hunk 1]
```diff
@@ -169,20 +168,12 @@ def __init__(
...
-    ]
 
     def _cleanup(self) -> None:
         # cleanup
         if self._delete_run_path and self._context is not None:
             for context_index in range(len(self._context)):
-                job_status = self._context.job_status(context_index)
-
-                if job_status == JobStatus.SUCCESS:
+                if self._context.is_job_completed(context_index):
```
Comment (xjules) [inline everest/suite.py:176]: Does it really check only for the job SUCCESS?

Comment (jonathan-eq) [reply everest/suite.py:176]: Yes. I think we should refactor everest away from using the internal ert states, and expose equivalent methods instead.

Comment (berland) [reply everest/suite.py:176]: it looks like Everest needs to know about waiting states, see `suite.py`

Comment (jonathan-eq) [reply everest/suite.py:176]: Yes, I updated both PRs to avoid this by exposing a `is_job_waiting()` method in BatchingContext instead

**Comment by jonathan-eq** [comment]
This is unblocked as ERT is no longer pinned to an older version

File: everest/version.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# file generated by setuptools_scm
+# don't change, don't track in version control
```
Comment (berland) [inline everest/version.py:2]: don't track in version control :)

**Review by berland — APPROVED** [review]
yay

**Pull request merged**